CameraOverlay/data.py

```diff
@@ -0,0 +1,111 @@
+import time
+from typing import Any, Optional
+import topics as topics
+
+V2_DATA_TOPICS = [
+    str(topics.DAS.data),
+    str(topics.PowerModel.recommended_sp),
+    str(topics.PowerModel.predicted_max_speed),
+    str(topics.PowerModel.plan_name),
+]
+
+V3_MESSAGE = [
+    str(topics.DAShboard.receive_message)
+]
+
+class Data:
+    """ A class to keep track of the most recent bike data for the overlays.
+        Data comes into the class in the V2/V3 MQTT formats and may be accessed
```

Comment:
This is a small nit, but can you make sure there is a blank line after the first sentence in the docstring (do it for every other method too)? It just makes it look neater. Eventually, I would love to have a linter for the docstrings.

Original:
<img width="633" alt="Screen Shot 2020-04-26 at 6 22 02 pm" src="https://user-images.githubusercontent.com/18709969/80302014-d4c50480-87ea-11ea-95a8-f93008c3ce7e.png">
Spacing:
<img width="666" alt="Screen Shot 2020-04-26 at 6 22 18 pm" src="https://user-images.githubusercontent.com/18709969/80302021-de4e6c80-87ea-11ea-8c27-8b23ee0b6be3.png">